### PR TITLE
Epayment upgrades

### PR DESCRIPTION
**Pull request by JPGarCar**
# TL;DR
Made changes to make the ubc_epyament app standalone from other apps. Only two things connect this app to others:
- The use of the Payment model
- Settings found on settings.py
This reduces coupling and makes it easier for others to use the ubc_epayment app.

# Description
- Removed the BasePayment model and now all the information is in the Payment model under the epayment app.
- epayment configuration settings like ID, password and code are user inputed as a Model called EPaymentConfig. There must be passed to a Payment model when created, usually stored by the products themselves.
- added success/denied/error HTML pages that are independent of other apps. They extend a parent template that is provided by the programmer in the settings.py file. After 5 seconds users are redirected to another page from another app, this page is also provided by the programmer in the settings.py
- epayment urls are now provided by the programmer in the settings.py file
- EPaymentConfig models can be set as in_testing which use the testing URL instead of the production URL
- Added a README file to explain how to use this app

## Why
Making this app standalone reduces the website's coupling dramatically! It also allows other programmers to copy the app without having to deal with other apps they might not have access to.

## User Changes
No user changes.
Administrators need to create EPaymentConfig objects and attach them to the products.

## Tests Done
Tested on localhost, need to test on server.

# Breaking Changes
Need to migrate changes! Payment model is different, will break code.

### CODE REVIEW

**Comment by JPGarCar** [comment]
Did testing on server and it works! Both success and declined.
